tbx/static_src/sass/components/_link-columns.scss

```diff
@@ -0,0 +1,48 @@
+@use 'config' as *;
+
+.link-columns {
+    &__header {
+        margin-bottom: $spacer-half;
+    }
+
+    &__links {
+        display: grid;
+        grid-template-columns: 1fr;
+        column-gap: $spacer-half;
+
+        @include media-query(medium) {
+            grid-template-columns: repeat(2, 1fr);
+        }
+
+        @include media-query(large) {
+            grid-template-columns: repeat(3, 1fr);
+        }
+    }
+
+    &__item {
+        padding: $spacer-mini-plus 0;
+        border-top: 1px solid var(--color--border);
+    }
+
+    &__link {
```

Comment:
I can see in the design that the links don't include underlines which goes away from the rest of the site where we've included links - it might be worth mentioning this to Ben E to see if he'd be up for adding underlines. 

If not we should add them back in when viewing in high contrast mode - there's a mixin you can use here: https://github.com/torchbox/torchbox.com/blob/main/tbx/static_src/sass/config/_mixins.scss#L243

Reply:
Hi @shyusu4 having links on underlines is something we've kept really consistent throughout the site, so although I don't have all the context, I'd be reluctant to make an exception here.

Reply:
Thank you @helenb, we've spoken to Ben about reconsidering this - will be updating this to include the underlines for links so it's consistent everywhere